fail2ban/tests/action_d/test_badips.py

```diff
@@ -153,5 +153,4 @@ def testBanIP(self):
 			# produce an error using wrong category/IP:
 			self.action._category = 'f2b-this-category-dont-available-test-suite-only'
 			aInfo['ip'] = ''
-			self.assertRaises(BadIPsActionTest.pythonModule.HTTPError, self.action.ban, aInfo)
```

Comment:
well, this whole removal is wrong (you want surely still call ban, and this assert is a wrapper catching error in callable;), rather you would replace it with something like:
```diff
-self.assertRaises(BadIPsActionTest.pythonModule.HTTPError, self.action.ban, aInfo)
+self.action.ban(aInfo)
```

Reply:
Yes you're right, thank you @sebres for your review and explanations, let's then test this :)

Reply:
Worked nicely, many thanks Sergey 👍